apps/src/lib/kits/maker/ui/DiscountAdminOverride.jsx

```diff
@@ -0,0 +1,141 @@
+import Button from "../../../../templates/Button";
+import ValidationStep, {Status} from '../../../../lib/ui/ValidationStep';
+
+const styles = {
+  title: {
+    fontSize: 32,
+  },
+};
+
+export default class EligibilityChecklist extends Component {
+  static propTypes = {
+    statusPD: PropTypes.oneOf(Object.values(Status)).isRequired,
+    statusStudentCount: PropTypes.oneOf(Object.values(Status)).isRequired,
+  };
+
+  state = {
+    statusYear: Status.UNKNOWN,
+    teacherID: "",
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      country: 'United States',
+      hoc: '',
+      nces: '',
+      schoolName: '',
+      schoolCity: '',
+      schoolState: '',
+      schoolZip: '',
+      schoolType: '',
+      afterSchool: '',
+      tenHours: '',
+      twentyHours: '',
+      otherCS: false,
+      followUpFrequency: '',
+      followUpMore: '',
+      acceptedPledge: false
+    },
+    errors: {
+      invalidEmail: false
+    }
+  };
+
+  handleDropdownChange = (field, event) => {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        [field]: event ? event.value : ''
+      }
+    });
+  }
+
+  handleSubmitId = () => {
+    this.setState({teacherID : this.teacherID.value});
+  }
+
+  render() {
+    const {submission, errors} = this.state;
+    return (
+      <div>
+        <h1 style={styles.title}>Circuit Playground Kits Admin Override</h1>
+        <form>
+          <label>
+            <div>Teacher email address or username (for the account they are using in the classroom):</div>
+            <input
+              value=""
+              ref={input => this.teacherID = input}
+            />
+            <Button
+              color="orange"
+              text="Submit"
+              onClick={this.handleSubmitId}
+            />
+          </label>
+        </form>
+          <div>
+            <h2>Eligibility requirements for </h2>
+            <ValidationStep
+              stepName={i18n.eligibilityReqPD()}
+              stepStatus={this.props.statusPD}
+            />
+            <ValidationStep
+              stepName={i18n.eligibilityReqStudentCount()}
+              stepStatus={this.props.statusStudentCount}
+            />
+            <ValidationStep
+              stepName={i18n.eligibilityReqYear()}
+              stepStatus={this.state.statusYear}
+            />
+            <h2>School Data</h2>
+            <SchoolAutocompleteDropdown
+              setField={this.handleDropdownChange}
+              value={submission.nces}
+              showErrorMsg={errors.nces}
+            />
+            <h2>Admin Options</h2>
+            <h4>Option 1: Link teacher account with other accounts</h4>
+            <div>
+              If teacher meets the eligibity requirements but is simply using a
+               different email address for their account than what we have on file,
+               please go to the Studio Person ID admin page to link this acccount
```

Comment:
I think spec calls for there to be a link to the page in here?

Reply:
Was waiting for the link. Just received and adding now.